API/app.py

```diff
@@ -34,30 +34,21 @@ def create_app():
     cursor = connection.cursor()
     print("CURSOR:", cursor)
 
-
+    # binding the instance to a very specific Flask app
+    # initialize app for use with this database setup
     db = SQLAlchemy(app)
     db.init_app(app)
 
+    # root route
     @app.route('/')
     def root():
         DB.create_all()
-        return "welcome to Med Cab"
-
-    @app.route('/predict', methods=['POST', 'GET'])
-    def root():
-        req_data = request.get_json(force=True)
-        output = predict(req_data)
-        return jsonify(output)
-
-    @app.route('/strains', methods=['POST'])
-    def strains():
+        return "Welcome to Med Cab"
 
-
-        return output
-
-    @app.route("/<test>", methods=['GET'])
-    def predict_strain(text=None):
+    # TODO: Make text value dynamic
+    @app.route("/test", methods=['GET'])
+    def predict_strain(text='I have fibromyalgia and I want pain relief'):
```

Comment:
the input should be a variable, like -- text --. We don't want the base input to be "I have fibromyalgia..."